main.go

```diff
@@ -74,6 +75,9 @@ func main() {
 	regionalSTS := flag.Bool("sts-regional-endpoint", false, "Whether to inject the AWS_STS_REGIONAL_ENDPOINTS=regional env var in mutated pods. Defaults to `false`.")
 	watchConfigMap := flag.Bool("watch-config-map", false, "Enables watching serviceaccounts that are configured through the pod-identity-webhook configmap instead of using annotations")
 	composeRoleArn := flag.Bool("compose-role-arn", false, "If true, then the role name and path can be used instead of the fully qualified ARN in the `role-arn` annotation.  In this case, webhook will look up the partition and account ID using instance metadata.  Defaults to `false`.")
+	watchConfigFile := flag.String("watch-config-file", "", "Absolute path to the config file to watch for")
+	containerCredentialsAudience := flag.String("container-credentials-audience", "pods.eks.amazonaws.com", "The audience for tokens used by the AWS Container Credentials method")
+	containerCredentialsFullUri := flag.String("container-credentials-full-uri", "http://169.254.170.23/v1/credentials", "AWS_CONTAINER_CREDENTIALS_FULL_URI will be set to this value in mutated containers")
```

Comment:
I think we need to be a bit smart about this URI. We need to prioritize calling the agent through its IPv6 endpoint (if supported) and if not, fallback to its IPv4 addr.

Reply:
Yes, either or.  When cluster is launch.  uri will be set to ipv6 if the cluster's IP Family is set to IPV6